core/chains/evm/client/errors.go

```diff
@@ -289,7 +290,7 @@ var internal = ClientErrors{
 	TerminallyStuck: regexp.MustCompile(TerminallyStuckMsg),
 }
 
-var clients = []ClientErrors{parity, geth, arbitrum, metis, substrate, avalanche, nethermind, harmony, besu, erigon, klaytn, celo, zkSync, zkEvm, treasure, mantle, aStar, hedera, gnosis, internal}
```

Comment:
Just to be clear, this `defaultClient` struct is used for something else so doesn't need to be a part of this list. Unfortunately, this file is a bit of a mess--at some point we should re-organize the whole thing to make more sense but that's out of scope ;-)

Reply:
+1 to both this probably doesn't need to be added here and that we probably should re-organize this better